modules/nw-about-configuring-master-interface-container.adoc

```diff
@@ -11,256 +11,4 @@ In {product-title} 4.14 and later, the ability to allow users to create a MAC-VL
-        "namespace": "test-namespace",
-        "interface": "ext0.100"
-      }
-    ]'
-spec:
-  securityContext:
-    runAsNonRoot: true
-  containers:
-    - name: nginx-container
-      image: nginxinc/nginx-unprivileged:latest
-      securityContext:
-        allowPrivilegeEscalation: false
-        capabilities:
-          drop: ["ALL"]
-      ports:
-        - containerPort: 80
-      seccompProfile:
-        type: "RuntimeDefault"
-----
-+
-<1> The name to be used as the master for the VLAN interface.
-
-. Apply the YAML by running the following command:
-+
-[source,terminal]
-----
-$ oc apply -f pod-a.yaml
-----
-
-. Get detailed information about the `nginx-pod` within the `test-namespace` by running the following command:
-+
-[source,terminal]
-----
-$ oc describe pods nginx-pod -n test-namespace
-----
-+
-.Expected output
-+
-[source,terminal]
-----
-Name:         nginx-pod
-Namespace:    test-namespace
-Priority:     0
-Node:         worker-1/10.46.186.105
-Start Time:   Mon, 14 Aug 2023 16:23:13 -0400
-Labels:       <none>
-Annotations:  k8s.ovn.org/pod-networks:
-                {"default":{"ip_addresses":["10.131.0.26/23"],"mac_address":"0a:58:0a:83:00:1a","gateway_ips":["10.131.0.1"],"routes":[{"dest":"10.128.0.0...
-              k8s.v1.cni.cncf.io/network-status:
-                [{
-                    "name": "ovn-kubernetes",
-                    "interface": "eth0",
-                    "ips": [
-                        "10.131.0.26"
-                    ],
-                    "mac": "0a:58:0a:83:00:1a",
-                    "default": true,
-                    "dns": {}
-                },{
-                    "name": "test-namespace/sriov-network",
-                    "interface": "ext0",
-                    "mac": "6e:a7:5e:3f:49:1b",
-                    "dns": {},
-                    "device-info": {
-                        "type": "pci",
-                        "version": "1.0.0",
-                        "pci": {
-                            "pci-address": "0000:d8:00.2"
-                        }
-                    }
-                },{
-                    "name": "test-namespace/vlan-100",
-                    "interface": "ext0.100",
-                    "ips": [
-                        "1.1.1.1"
-                    ],
-                    "mac": "6e:a7:5e:3f:49:1b",
-                    "dns": {}
-                }]
-              k8s.v1.cni.cncf.io/networks:
-                [ { "name": "sriov-network", "namespace": "test-namespace", "interface": "ext0" }, { "name": "vlan-100", "namespace": "test-namespace", "i...
-              openshift.io/scc: privileged
-Status:       Running
-IP:           10.131.0.26
-IPs:
-  IP:  10.131.0.26
-----
\ No newline at end of file
+To ensure the use of a container namespace master interface specify the `linkInContainer` and set the value to `true` in the VLAN, MACVLAN, or IPVLAN plugin configuration depending on the particular type of additional network.
```

Comment:
While no rules a comma necessary, I think that adding one might help with readability. Lot of words there. 
```suggestion
To ensure the use of a container namespace master interface, specify the `linkInContainer` and set the value to `true` in the VLAN, MACVLAN, or IPVLAN plugin configuration depending on the particular type of additional network.
```